spyt-package/src/main/python/spyt/utils.py

```diff
@@ -262,8 +262,9 @@ def set_conf(conf, dict_conf):
             conf.set(key, value)
 
 
-def default_discovery_dir():
-    return os.getenv("SPARK_YT_DISCOVERY_DIR") or YPath("//home").join(os.getenv("USER")).join("spark-tmp")
+def default_discovery_dir(client=None):
+    return os.getenv("SPARK_YT_DISCOVERY_DIR") \
+        or YPath("//home").join(default_user(client=client)).join("spark-tmp")
```

Comment:
Let's change this to //tmp? None of the known YT clusters allows having top-level home user dirs.